synchronization/src/androidTest/java/de/cyface/synchronization/WifiSurveyorTest.java

```diff
@@ -97,6 +98,7 @@ public void tearDown() {
      * This test may be flaky on a read device when the network changes during the test.
      */
     @Test
+    @Ignore // TODO [MOV-644]: flaky
```

Comment:
I've this task "in doing" but postponed because of the open priority bugs. This way I don't have to execute the CI build 5+ times until this test goes through -.-